src/eth-utils.js

```diff
@@ -1,5 +1,61 @@
 import { firmwareAPI, HARDENED } from './bitbox02';
 
+const getCoinFromChainId = chainId => {
+    switch(chainId) {
+        case 1:
+            return firmwareAPI.messages.ETHCoin.ETH;
+        case 3:
+            return firmwareAPI.messages.ETHCoin.RopstenETH;
+        case 4:
+            return firmwareAPI.messages.ETHCoin.RinkebyETH;
+        default:
+            throw new Error('Unsupported network');
+    }
+}
+
+/**
+ * @param keypathString keypath in string format e.g. m/44'/1'/0'/0
+ * @returns keypath as array e.g. [2147483692, 2147483649, 2147483648, 0]
+ */
+export const getKeypathFromString = keypathString => {
+    let levels = keypathString.toLowerCase().split('/');
+    if (levels[0] !== 'm') throw new Error('Invalid keypath');
+    levels = levels.slice(1, levels.length);
```

Comment:
nit: same as just `levels.splice(1)`, which deletes the first element without making a copy of the rest.

Reply:
Good point.